Dockerfile

```diff
@@ -2,6 +2,9 @@ FROM openresty/openresty:alpine-fat
 
 RUN /usr/local/openresty/luajit/bin/luarocks install lua-resty-template
 RUN /usr/local/openresty/luajit/bin/luarocks install lua-resty-http
+RUN /usr/local/openresty/luajit/bin/luarocks install busted
+RUN /usr/local/openresty/luajit/bin/luarocks install lrexlib-posix
```

Comment:
Needed for testing regex's in lua as string.match does not support some special characters like `|` for example.